plugins/wasm-go/extensions/ai-search/main.go

```diff
@@ -24,20 +23,18 @@ import (
 	"time"
 	"unicode"
 
-	"github.com/higress-group/proxy-wasm-go-sdk/proxywasm"
-	"github.com/higress-group/proxy-wasm-go-sdk/proxywasm/types"
-	"github.com/higress-group/wasm-go/pkg/log"
-	"github.com/tidwall/gjson"
-	"github.com/tidwall/sjson"
-
-	"github.com/higress-group/wasm-go/pkg/wrapper"
-
 	"github.com/alibaba/higress/plugins/wasm-go/extensions/ai-search/engine"
 	"github.com/alibaba/higress/plugins/wasm-go/extensions/ai-search/engine/arxiv"
 	"github.com/alibaba/higress/plugins/wasm-go/extensions/ai-search/engine/bing"
 	"github.com/alibaba/higress/plugins/wasm-go/extensions/ai-search/engine/elasticsearch"
 	"github.com/alibaba/higress/plugins/wasm-go/extensions/ai-search/engine/google"
 	"github.com/alibaba/higress/plugins/wasm-go/extensions/ai-search/engine/quark"
+	"github.com/higress-group/proxy-wasm-go-sdk/proxywasm"
+	"github.com/higress-group/proxy-wasm-go-sdk/proxywasm/types"
```

Comment:
The package import order is incorrect; external packages should be placed at the beginning